packages/shared/lib/core/network/utils/getDefaultPersistedNetwork.ts

```diff
@@ -1,20 +1,13 @@
-import { DEFAULT_CHAIN_CONFIGURATIONS, DEFAULT_NETWORK_METADATA } from '../constants'
+import { DEFAULT_NETWORK_METADATA } from '../constants'
 import { NetworkId } from '../enums'
 import { IPersistedNetwork } from '../interfaces'
 
 export function getDefaultPersistedNetwork(networkId: NetworkId): IPersistedNetwork {
     const network = DEFAULT_NETWORK_METADATA?.[networkId]
-    const configuration = DEFAULT_CHAIN_CONFIGURATIONS?.[networkId]
```

Comment:
I would probably not remove this so we dont lose the l1->l2 functionality

Reply:
Re-added the deleted export and code, check if this is what you meant. 😅